packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -11,26 +11,27 @@
  */
 
 .root {
-  @include border-radius($base-border-radius);
+  @include border-radius(var(--BorderRadius));
 
   display: inline-block;
   position: relative;
-  margin-block: 0;
-  margin-inline: 0;
-  padding-block: $spacing-xs-em;
-  padding-inline: $spacing-s-em;
+  margin-block: var(--SpaceMarginBlock);
+  margin-inline: var(--SpaceMarginInline);
+  padding-block: var(--SpacePaddingBlock);
+  padding-inline: var(--SpacePaddingInline);
   transition-property: color, background-color, border-color, box-shadow;
-  transition-duration: 100ms;
-  transition-timing-function: linear;
-  border: 1px solid transparent;
+  transition-duration: var(--TransitionDuration);
+  transition-timing-function: var(--TransitionTiming);
+  border-width: var(--BorderWidth);
+  border-style: var(--BorderStyle);
+  border-color: var(--ColorBorderPrimary);
   background-color: var(--ColorBackgroundPrimary);
-  box-shadow: 0 0 0 0 $color-primary-light;
-  color: $white;
-  font-family: $body-font-family;
-  font-size: ms(0);
-  font-weight: 600;
-  line-height: $base-line-height;
-  white-space: nowrap;
+  color: var(--ColorFont);
+  font-family: var(--FontFamily);
+  font-size: var(--FontSize);
+  font-weight: var(--FontWeight);
+  line-height: var(--FontLineHeight);
+  white-space: var(--FontWhiteSpace);
```

Comment:
I don't think `white-space` is intended to be adjustable here since Buttons should never have wrapping text.

Reply:
https://okta.slack.com/archives/C0223BW80E9/p1642610245017400